src/photos/components/Topbar.jsx

```diff
@@ -66,9 +66,9 @@ EditableAlbumName.propTypes = {
 }
 
 const TopbarTitle = ({ children }) => (
-  <h2 data-test-id="pho-content-title" className={styles['pho-content-title']}>
+  <AppTitle data-test-id="pho-content-title" className="pho-content-title">
```

Comment:
`className="pho-content-title"` is probably wrong though, we should use the css module version?

Reply:
Yeah but… I did… What's going on here? 🤔